mkosi/distributions/gentoo.py

```diff
@@ -1,198 +1,99 @@
 # SPDX-License-Identifier: LGPL-2.1+
 
-import logging
 import os
 import re
 import urllib.parse
 import urllib.request
 from collections.abc import Sequence
 from pathlib import Path
 from textwrap import dedent
+from typing import Optional
 
 from mkosi.architecture import Architecture
 from mkosi.distributions import DistributionInstaller
-from mkosi.install import copy_path, flock
+from mkosi.install import copy_path
 from mkosi.log import ARG_DEBUG, complete_step, die, log_step
 from mkosi.remove import unlink_try_hard
 from mkosi.run import run, run_workspace_command
 from mkosi.state import MkosiState
+from mkosi.types import PathString
 
 
 def invoke_emerge(
     state: MkosiState,
+    sysroot: Optional[Path] = None,
```

Comment:
I don't see `invoke_emerge()` being called anywhere with `sysroot` set to `None` so this doesn't need to be optional?

Reply:
This was not addressed?

Reply:
removed the `Optional`.